lib/src/install/osconfig.rs

```diff
@@ -18,6 +23,14 @@ pub(crate) fn inject_root_ssh_authorized_keys(root: &Dir, contents: &str) -> Res
     root.create_dir_all(tmpfiles_dir)?;
     let target = tmpfiles_dir.join(ROOT_SSH_TMPFILE);
     root.atomic_write(&target, &tmpfiles_content)?;
+
+    let as_path = Utf8Path::new(ETC_TMPFILES).join(ROOT_SSH_TMPFILE);
+    state.lsm_label(
+        &Utf8PathBuf::from("/").join(root_path).join(&as_path),
+        &Utf8PathBuf::from("/").join(&as_path),
```

Comment:
I think this is just `root_path.join(as_path)` and `as_path` respectively, right?  They're already absolute paths I believe.

Reply:
The second one is actually relative, I discovered while writing the updated test. :)

Reply:
Hm, I now see one Ci failure:
```
 failed to execute cmd-kola: exit status 1
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:84","func":"k8s.io/test-infra/prow/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2024-03-15T12:50:22Z"}
error: failed to execute wrapped command: exit status 1 
```
but cannot see more details, I wonder if it's related or not to my changes :/